dash-pipeline/dockerfiles/Dockerfile.p4c-bldr

```diff
@@ -0,0 +1,111 @@
+# https://hub.docker.com/r/p4lang/behavioral-model
+FROM p4lang/behavioral-model:no-pi
+LABEL maintainer="SONIC-DASH Community"
+LABEL description="DASH pipeline P4 Behavioral Model and toolchain"
+
+# Configure make to run as many parallel jobs as cores available
+ARG available_processors
+ARG MAKEFLAGS=-j$available_processors
+
+
+# Select the compiler to use.
+# We install the default version of GCC (GCC 9), as well as clang 8 and clang 10.
+ARG sswitch_grpc=yes
+ARG CC=gcc
+ARG CXX=g++
+ENV TZ=America/Los_Angeles
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+ENV GIT_SSL_NO_VERIFY=true
+
+ENV PI_DEPS automake \
+            build-essential \
+            clang-8 \
+            clang-10 \
+            clang-format-8 \
+            g++ \
+            libboost-dev \
+            libboost-system-dev \
+            libboost-thread-dev \
+            libtool \
+            libtool-bin \
+            pkg-config \
+            libreadline-dev \
+            libpcap-dev \
+            libmicrohttpd-dev \
+            doxygen \
+            valgrind \
+            vim \
+            git-core \
+            python3 \
+            cmake \
+            python3-pip
+
+ENV PI_RUNTIME_DEPS libboost-system1.71.0 \
+                    libboost-thread1.71.0 \
+                    libpcap0.8 \
+                    python3 \
+                    python-is-python3
+
+RUN apt-get update && apt-get install -y --no-install-recommends $PI_DEPS $PI_RUNTIME_DEPS
+
+RUN cd / && git clone --depth=1 -b v1.43.2 https://github.com/google/grpc.git && \
```

Comment:
Instead of building gRPC, try using the technique here to use pre-built grpc libraries, it's much faster:
https://github.com/Azure/DASH/blob/main/dash-pipeline/dockerfiles/Dockerfile.bmv2-bldr#L3
https://github.com/Azure/DASH/blob/main/dash-pipeline/dockerfiles/Dockerfile.bmv2-bldr#L54

Reply:
Done